lib/pacio_test_kit/common_tests/update_test.rb

```diff
@@ -0,0 +1,52 @@
+require_relative '../pacio_profiles'
+require_relative '../interactions_test'
+
+module PacioTestKit
+  class UpdateTest < Inferno::Test
+    include PacioTestKit::PacioProfiles
+    include PacioTestKit::InteractionsTest
+
+    title 'Resources returned in previous read tests can be updated'
+    id :pacio_resource_update
+    description %(
+      This test verifies resources returned from the read tests are able to be updated.
+
+      This test will be skipped if no resource read requests were made in
+      previous tests or if all requests were unsuccessful.
+    )
+
+    input :id_to_update,
+          title: 'The id of the FHIR resource to update on the server',
+          description: 'Provide a string id of a resource to update on the server.'
+
+    input :new_resource,
+          title: 'The resource to update the FHIR resource on the server',
+          description: 'Provide a FHIR resource to update on the server.'
```

Comment:
Do not need these inputs. The resource to update will be obtained from previous read or search test. You should just consider read test for now as search test has not yet been implemented. Inferno will be attempting to update an existing resource, so no need for the user's input. You can try to update something simple like the `status` element.